### PR TITLE
fix: Stabilize send kudos and open kudos list drawer - MEED-2778 - Meeds-io/MIPs#81

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: src/main/java/io/meeds/qa/ui/pages/SpaceHomePage.java [hunk 1]
```diff
@@ -1780,7 +1780,7 @@ private ElementFacade kudosButtonNumberToCommentActivityElement() {
   }
 
   private ElementFacade kudosButtonToCommentActivityElement() {
-    return findByXPathOrCSS("//div[contains(@class,'white border-radius')][1]//div[contains(@class,'v-list flex')]//div[contains(@class,'d-inline-flex')][1]//div[@role='button']//span[@class='v-btn__content'][last()]");
+    return findByXPathOrCSS("//div[contains(@class,'white card-border-radius')][1]//div[contains(@class,'v-list flex')]//div[contains(@class,'d-inline-flex')][1]//div[@role='button']//span[@class='v-btn__content'][last()]");
```
Comment (boubaker) [inline src/main/java/io/meeds/qa/ui/pages/SpaceHomePage.java]: Let's attempt to fix this CSS selector and make it more generic and stable.

File: src/test/resources/features/Social/ActivityStream.feature [hunk 2]
```diff
@@ -1527,7 +1527,7 @@ Feature: Activity Stream
     Then I click on update comment
     And I open in activity 'activitycap88' the Comments drawer
     Then Comment is displayed in comments drawer at the sixth position
-
+
```
Comment (boubaker) [inline src/test/resources/features/Social/ActivityStream.feature]:
```suggestion

```

File: src/main/java/io/meeds/qa/ui/pages/SpaceHomePage.java (hunk 1 above)
Comment (SaraBoutej) [reply src/main/java/io/meeds/qa/ui/pages/SpaceHomePage.java]: done

File: src/main/java/io/meeds/qa/ui/pages/SpaceHomePage.java [hunk 3]
```diff
@@ -1780,7 +1780,7 @@ private ElementFacade kudosButtonNumberToCommentActivityElement() {
   }
 
   private ElementFacade kudosButtonToCommentActivityElement() {
-    return findByXPathOrCSS("//div[contains(@class,'white border-radius')][1]//div[contains(@class,'v-list flex')]//div[contains(@class,'d-inline-flex')][1]//div[@role='button']//span[@class='v-btn__content'][last()]");
+    return findByXPathOrCSS("//div[contains(@class,'unread-activity')][1]//div[contains(@class,'v-list flex')]//*[contains(@id, 'ActivityCommment_comment')]//div[contains(@class,'d-inline-flex')][1]//div[@role='button']//span[@class='v-btn__content'][last()]");
```
Comment (boubaker) [inline src/main/java/io/meeds/qa/ui/pages/SpaceHomePage.java]: We should have something similar to what was made in method `getActivityFooterButton`

```
    return getCommentFooterButton(comment, "KudosActivity");
```

File: src/test/resources/features/Social/ActivityStream.feature [hunk 4]
```diff
@@ -1528,6 +1528,7 @@ Feature: Activity Stream
     And I open in activity 'activitycap88' the Comments drawer
     Then Comment is displayed in comments drawer at the sixth position
 
+    @test
```
Comment (boubaker) [inline src/test/resources/features/Social/ActivityStream.feature]:
```suggestion
```

File: src/test/resources/features/Social/ActivityStream.feature [hunk 5]
```diff
@@ -1781,6 +1782,7 @@ Feature: Activity Stream
     And I open in activity 'activitycap87' the Comments drawer
     Then Fourth comment is displayed in comments drawer
 
+    @test
```
Comment (boubaker) [inline src/test/resources/features/Social/ActivityStream.feature]:
```suggestion
```

**Review by boubaker — COMMENTED** [review]
ok for me, just need to delete the `@test` tag after the tests finishes its execution on the CI
Thanks @SaraBoutej

**Comment by SaraBoutej** [comment]
> ok for me, just need to delete the `@test` tag after the tests finishes its execution on the CI Thanks @SaraBoutej

removed after test success on the CI 
https://ci.exoplatform.org/job/exo-qa-ui/job/qa-ui-auto-meeds/665/Serenity/

**Review by boubaker — APPROVED** [review]

**Pull request merged**